src/theme.js

```diff
@@ -2,13 +2,19 @@ import { createTheme } from '@mui/material/styles';
 
 const theme = createTheme({
   typography: {
-    fontFamily: 'Roboto, sans-serif'
+    fontFamily: 'Roboto, sans-serif',
+    dd: { fontWeight: 'bold' },
```

Comment:
Ok, so there is a "dd" in the theme - man, I'm sorry. Jumped the gun on some comments in other files. 

BUT, the font doesn't actually display as bold - so this is probably not implemented correctly.

Reply:
Thinking of replacing it with just sx locally, think it'll be better than setting custom variants since the main point of the status message is just to indicate to the user what's happening.